### ISSUE TITLE
Aside must not be contained in another landmark

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwenty#703]
Aside (with role="complementary") , which is used to hold the [footer widgets](https://github.com/WordPress/twentytwenty/blob/master/template-parts/footer-menus-widgets.php#L95) cannot be inside the `<main>`, it needs to be at top level, not nested.

**Comment by glauberm** [comment]
Good one. I'll make a PR addressing it in a few minutes.

**Comment by joyously** [comment]
Wasn't there already an issue (and PR) for this?

**Comment by carolinan** [comment]
Maybe you are thinking of the issue with the footer landmark?

**Comment by glauberm** [comment]
Could the solution be as simple as this: #704 ? Testing locally, everything is okay.
